front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -127,33 +150,14 @@ export default function Study() {
     setSortOption(e.target.value as SortOptions);
   };
 
-  const getProject = async () => {
-    if (currentProject?.id) {
-      try {
-        setProject(currentProject);
-        await getStudyStates();
-      } catch (error) {
-        console.error(error);
-      }
-    }
-  };
-
   const getStudy = async (withNotification = false) => {
-    if (projectId && studyId) {
-      try {
-        const { data } = await getCurrentStudy({ projectId, studyId });
-        if (data) setStudy(data as StudyWithFileType);
-        if (withNotification) {
-          dispatch(
-            setSuccess({
-              title: t('studyUpdated'),
-              text: t('studyUpdatedDetails', { name: study?.name }),
-            })
-          );
-        }
-      } catch (error) {
-        console.error(error);
-      }
+    if (withNotification) {
+      dispatch(
+        setSuccess({
+          title: t('studyUpdated'),
+          text: t('studyUpdatedDetails', { name: study?.name }),
+        })
+      );
```

Comment:
```suggestion
```
You can totally remove this function, and move the notification directly in the `AddOrEditStudyModal`, in `updateStudy` :)